src/simplnx/DataStructure/IDataStore.hpp

```diff
@@ -73,6 +73,15 @@ class SIMPLNX_EXPORT IDataStore
     return getNumberOfTuples() * getNumberOfComponents();
   }
 
+  /**
+   * @brief Returns the number of values stored within the DataStore.
+   * @return usize
+   */
+  usize size() const
+  {
+    return getNumberOfTuples() * getNumberOfComponents();
```

Comment:
```suggestion
    return getSize();
```
We can implement `size()` using `getSize()`.